### DISCUSSION TITLE
track edited messages

### DISCUSSION THREAD

**Discussion by bobi808** [tomer8007/whatsapp-web-incognito#141] [Ideas]
plz add an option to track edited messages

**Comment by tomer8007** [comment]
Hi, this was discussed in #126. Saving history is easy. The UI is the annoying thing for me to do.

**Reply by bobi808** [reply]
It could take all edited message and show them in a single box under neath for example

**Comment by bobi808** [comment]
It could take all edited message and show them in a single box under neath for example

**Reply by tomer8007** [reply]
Pretty complex to implement considering you have video messages, voice messages, etc. The solution has to involve feeding WhatsApp with the original message data, but still, creating this GUI and finding the right places to call will take some work.

**Comment by tomer8007** [comment]
I think the best GUI will be a feature that lets you go back and forward in the message versions. This way I just need to find the right function to call and stuff around that

**Reply by bobi808** [reply]
That's a good idea, however i think only text messages could be edited and there is no worry about other kind of messages.

**Reply by bobi808** [reply]
Hope you will find a way to add this feature soon 🫡🌹
